app/src/features/admin/users/access-requests/list/pending/AccessRequestPendingList.tsx

```diff
@@ -33,7 +32,10 @@ interface IAccessRequestPendingListProps {
  * @returns
  */
 const AccessRequestPendingList = (props: IAccessRequestPendingListProps) => {
-  const { accessRequests, codes, refresh } = props;
+  const { accessRequests, refresh } = props;
+
+  const codesContext = useCodesContext();
```

Comment:
Will want to add a useEffect to ensure codes are loaded:

```typescript
  useEffect(() => {
    codesContext.codesDataLoader.load();
  }, [codesContext.codesDataLoader]);
```

Reply:
(TBH, I've been wondering about just putting the useEffect in the `useCodesContext` hook, that way no one ever has to think about it. But, something for another PR.)

Reply:
Fixed